DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/PresetServlet.java

```diff
@@ -51,11 +50,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String fromDate = request.getParameter("start_date");
     String toDate = request.getParameter("end_date");
     double dailyBudget = Double.parseDouble(request.getParameter("daily_budget"));
-    String location = request.getParameter("location");
-    String language = request.getParameter("language");
+    String location = request.getParameter("locations");
     String domain = request.getParameter("domain");
-    String target = request.getParameter("target_page");
-    String adText = request.getParameter("ad_text");
+    String target = request.getParameter("targets");
+    String adText = request.getParameter("adText");
     double cpc = Double.parseDouble(request.getParameter("cpc"));
```

Comment:
nit: Can you change cpc to manualCPC to match the variable names I was using? Also, why aren't you getting the name of the campaign as well?